lib/src/settings_tile.dart

```diff
@@ -101,25 +101,35 @@ class SettingsTile extends StatelessWidget {
 
   Widget androidTile() {
     if (_tileType == _SettingsTileType.switchTile) {
-      return SwitchListTile(
-        secondary: leading,
-        value: switchValue,
-        activeColor: switchActiveColor,
-        onChanged: enabled ? onToggle : null,
-        title: Text(title, style: titleTextStyle),
-        subtitle:
-            subtitle != null ? Text(subtitle, style: subtitleTextStyle) : null,
-        inactiveTrackColor: inactiveTrackColor,
-      );
+      return InkWell(
+          onTap: onTap,
+          focusColor: Colors.white24,
+          child: SwitchListTile(
+            secondary: leading,
+            value: switchValue,
+            activeColor: switchActiveColor,
+            onChanged: enabled ? onToggle : null,
+            title: Text(title, style: titleTextStyle),
+            subtitle: subtitle != null ? Text(
+                subtitle, style: subtitleTextStyle) : null,
+            inactiveTrackColor: inactiveTrackColor,
+          ));
     } else {
-      return ListTile(
-        title: Text(title, style: titleTextStyle),
-        subtitle:
-            subtitle != null ? Text(subtitle, style: subtitleTextStyle) : null,
-        leading: leading,
-        enabled: enabled,
-        trailing: trailing,
+      return InkWell(
         onTap: onTap,
+        focusColor: Colors.white24,
```

Comment:
@jja08111 can you remove all hardcoded colors? If the programmer has a customized theme, then this will prevent him from achieving a UI requirements.

Reply:
This is already deleted [this commit](https://github.com/yako-dev/flutter-settings-ui/pull/39/commits/7d5ec265bacebc022ff963c1efdd8123b209f0eb).